mods/cnc/mod.yaml

```diff
@@ -153,7 +153,8 @@ ContentInstaller:
 		.: conquer.mix, desert.mix, general.mix, scores.mix, sounds.mix, temperat.mix, winter.mix
 	ShippedSoundtracks: 4
 	MusicPackageMirrorList: http://www.openra.net/packages/cnc-music-mirrors.txt
-	InstallShieldCABFileIds: 66, 45, 42, 65, 68, 67, 71, 47, 49, 60, 75, 73, 53
+	InstallShieldPackageName: CnC
+	ExtractFromInstallShieldPackage: SPEECH.MIX, CONQUER.MIX, CCLOCAL.MIX, SOUNDS.MIX, TEMPICNH.MIX, TEMPERAT.MIX, TRANSIT.MIX, DESERT.MIX, GENERAL.MIX, SCORES.MIX, WINTER.MIX, UPDATEC.MIX
```

Comment:
why we have to test this?! the filenames will always lowercase

https://github.com/LipkeGu/OpenRA/commit/54f878993b02c2d8e3ddcd98ba01dcd74734c056#diff-d7d7b22ed1d8f9159c5bc9aa83fdb4d0R437